<launch>

    <arg name="map_size_x" default="20.0"/>
    <arg name="map_size_y" default="12.0"/>
    <arg name="map_size_z" default="2.5"/>

    <arg name="max_vel" default="1.0" />
    <arg name="max_acc" default="1.0" />

    <arg name="resolution" default="0.05"/>
    <arg name="poly_order" default=" 5"  />

    <param name="/odom_visualization/pub_path_flag"  type="bool"  value="false"/>

    <node name="$(anon rviz)" pkg="rviz" type="rviz" respawn="false"
          args="-d $(find interactive_marker_tutorials)/launch/rviz_config/default.rviz" output="screen">
    </node>

    <node pkg="interactive_marker_tutorials" type="airborne_control" name="airborne_control" output="screen">
        <param name="start_x" value="0"/>
        <param name="start_y" value="0"/>
        <param name="start_z" value="1"/>
        <param name="x_range_min" value="-10"/>
        <param name="y_range_min" value="-5"/>
        <param name="z_range_min" value="0.5"/>
        <param name="x_range_max" value="20"/>
        <param name="y_range_max" value="20"/>
        <param name="z_range_max" value="3"/>
        <param name="mesh_marker_flag" type="bool" value="false"/>
    </node>

    <!--<node pkg="interactive_marker_tutorials" type="basic_controls" name="basic_controls" output="screen"/>-->


</launch>
